src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,23 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.12.2
+
+This is a bugfix release:
+
+* Globalnet metrics port will now be opened by default when Globalnet is deployed.
+* Submariner ServiceExport now has unique condition types to wait for readiness.
+* The `subctl diagnose` command now supports NAT-discovery port validation.
+* The `cloud prepare` command will now work properly for nodes to which security groups were added manually.
```

Comment:
```suggestion
* The `subctl cloud-prepare` command will now work properly for nodes to which security groups were added manually.
```

Reply:
I think it should be 'subctl cloud prepare'

Reply:
Oops, sorry!